### PR TITLE
Improve code readability.

### PR DESCRIPTION
**Pull request by Comolli**
All the inflights_test code passed.

### CODE REVIEW

File: raft/tracker/inflights.go [hunk 1]
```diff
@@ -56,11 +56,9 @@ func (in *Inflights) Add(inflight uint64) {
 	if in.Full() {
 		panic("cannot add into a Full inflights")
 	}
-	next := in.start + in.count
-	size := in.size
-	if next >= size {
-		next -= size
-	}
+
+	next := (in.start + in.count) % in.size
```
Comment (SimFG) [inline raft/tracker/inflights.go:60]: If `in.start + in.count` is more than twice as large as `in.size`, `next` will be changed compared to before. And the `next` will decide whether to execute `grow`. Maybe some context I don't know.

**Review by spzala — COMMENTED** [review]
@Comolli thanks but I think it's unnecessary changes to stable raft library. Please continue helping project with work on open issues or other improvements. Thanks! cc @ahrtr

**Comment by spzala** [comment]
@Comolli thanks again. Closing it per discussion here. Please continue helping project with work on open issues or other improvements.

File: raft/tracker/inflights.go (hunk 1 above)
Comment (Comolli) [reply raft/tracker/inflights.go:60]: Thank you for your kind reply.

**Pull request closed without merging**